nextroute/schema/fleet.go

```diff
@@ -0,0 +1,234 @@
+// Package schema provides the input and output schema for nextroute.
+package schema
+
+import (
+	"fmt"
+	"time"
+)
+
+// FleetInput schema.
+type FleetInput struct {
```

Comment:
```suggestion
// FleetInput schema.
// DEPRECATION NOTICE: this part of the API is deprecated and is no longer maintained. It will be deleted soon. Please use [Input] instead.
type FleetInput struct {
```

For good measure let's make sure anyone seeing this will know it is deprecated.

Reply:
sure